woningwaardering/stelsels/zelfstandige_woonruimten/gemeenschappelijke_parkeerruimten/gemeenschappelijke_parkeerruimten.py

```diff
@@ -0,0 +1,219 @@
+)
+from woningwaardering.vera.referentiedata import (
+    Woningwaarderingstelsel,
+    Woningwaarderingstelselgroep,
+)
+from woningwaardering.vera.referentiedata.bouwkundigelementdetailsoort import (
+    Bouwkundigelementdetailsoort,
+)
+from woningwaardering.vera.referentiedata.ruimtedetailsoort import Ruimtedetailsoort
+from woningwaardering.vera.utils import heeft_bouwkundig_element
+
+
+class GemeenschappelijkeParkeerruimten(Stelselgroep):
+    parkeertype_punten_mapping = {
+        Ruimtedetailsoort.parkeervak_auto_binnen.code: {"Type I": Decimal("9.0")},
+        Ruimtedetailsoort.carport.code: {"Type II": Decimal("6.0")},
+        Ruimtedetailsoort.parkeervak_auto_buiten_niet_overdekt.code: {
+            "Type III": Decimal("4.0")
+        },
+    }
+
+    def __init__(
+        self,
+        peildatum: date = date.today(),
+    ) -> None:
+        self.stelsel = Woningwaarderingstelsel.zelfstandige_woonruimten
+        self.stelselgroep = (
+            Woningwaarderingstelselgroep.gemeenschappelijke_parkeerruimten
+        )
+        super().__init__(
+            begindatum=date.fromisoformat("2024-07-01"),
+            einddatum=date.max,
+            peildatum=peildatum,
+        )
+
+    def bereken(
+        self,
+        eenheid: EenhedenEenheid,
+        woningwaardering_resultaat: (
+            WoningwaarderingResultatenWoningwaarderingResultaat | None
+        ) = None,
+    ) -> WoningwaarderingResultatenWoningwaarderingGroep:
+        woningwaardering_groep = WoningwaarderingResultatenWoningwaarderingGroep(
+            criteriumGroep=WoningwaarderingResultatenWoningwaarderingCriteriumGroep(
+                stelsel=Woningwaarderingstelsel.zelfstandige_woonruimten.value,
+                stelselgroep=Woningwaarderingstelselgroep.gemeenschappelijke_parkeerruimten.value,
+            )
+        )
+
+        woningwaardering_groep.woningwaarderingen = []
+
+        if eenheid.ruimten is None:
+            warnings.warn(f"Eenheid {eenheid.id} heeft geen 'ruimten'")
+            return woningwaardering_groep
+
+        for ruimte in eenheid.ruimten:
+            if ruimte.detail_soort is None or ruimte.detail_soort.code is None:
+                warnings.warn(
+                    f"Ruimte {ruimte.id} heeft geen 'detail_soort' of 'detail_soort.code'"
+                )
+                continue
+
+            # Het beleidsboek geeft niet aan hoe er punten gegeven moeten worden voor onderstaande ruimtes
+            if ruimte.detail_soort.code in [
+                Ruimtedetailsoort.parkeervak_motorfiets_binnen.code,
+                Ruimtedetailsoort.parkeervak_scootmobiel_binnen.code,
+                Ruimtedetailsoort.stalling_extern.code,
+                Ruimtedetailsoort.stalling_intern.code,
+                Ruimtedetailsoort.parkeervak_motorfiets_buiten_niet_overdekt.code,
+                Ruimtedetailsoort.parkeervak_scootmobiel_buiten.code,
+            ]:
+                logger.warning(
```

Comment:
```suggestion
                logger.info(
```
opzich is dit niet zo spannend denk ik?